services/tests/endpoint_tests/test_department_endpoint.py

```diff
@@ -0,0 +1,58 @@
+from uuid import UUID
+
+import pytest  # noqa: F401;
+
+from rest_framework.reverse import reverse
+
+from services.tests.endpoint_tests.fixtures import *  # noqa: F403, F401;
+from services.tests.utils import get
+
+deps = [UUID('da792f32-6da7-4804-8059-16491b1ec0fa'), UUID('92f9182e-0942-4d82-8b6a-09499fe9c46a'),
```

Comment:
This list, and the list in fixtures.departments, should be derived from the same set of source values to avoid manual sync.